src/panels/config/person/ha-config-person.ts

```diff
@@ -228,15 +247,22 @@ class HaConfigPerson extends LitElement {
         margin: 16px auto;
         overflow: hidden;
       }
+      .picture {
+        width: 40px;
+        height: 40px;
+        background-size: cover;
+        border-radius: 50%;
+        background-position: center;
```

Comment:
I like this, but it is different behavior than a badge has, I think we should align that